src/main/resources/interface-specification.yml

```diff
@@ -2519,6 +2702,91 @@ components:
       schema:
         type: string
   schemas:
+    Mail:
+      type: object
+      required:
+        - address
```

Comment:
I know you know, but 
```suggestion
      required:
        - address
        - description
```

Reply:
The requirements were to have description optional